man/udevadm.xml

```diff
@@ -788,6 +788,14 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><option>--no-style</option></term>
+          <listitem>
+            <para>Ignore style issues. When specified, even if style issues found in a udev rules file, the
+            exit status is <constant>0</constant> unless other syntactic or semantic errors found.</para>
```

Comment:
"the exit status is 0 if no syntactic or semantic errors are found"

Reply:
Yes, there should be "are found" in both cases.